lib/checkassert.cpp

```diff
@@ -39,118 +40,172 @@ namespace {
     CheckAssert instance;
 }
 
-void CheckAssert::assertWithSideEffects()
-{
+void CheckAssert::assertWithSideEffects() {
     if (!mSettings->severity.isEnabled(Severity::warning))
         return;
 
-    for (const Token* tok = mTokenizer->list.front(); tok; tok = tok->next()) {
```

Comment:
could you remove such stylistic changes? I am not strongly against this but it's easier to review if the PR only shows the relevant changes.

Reply:
I tried my best to not change the style but if you look at line 51 and 52 the style is even mixed in this file only. So I looked at different files and created a clangformat configuration that fit best.

Reply:
if you want to use a formatter.. we use uncrustify and there is a script runformat in the root folder that formats.